### PR TITLE
Add example of the service account auth

### PR DESCRIPTION
**Pull request by tmotyl**
Fixes: https://github.com/iterative/PyDrive2/issues/21

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -29,11 +30,47 @@ Create *quickstart.py* file and copy and paste the following code.
...
     gauth.LocalWebserverAuth() # Creates local webserver and auto handles authentication.
 
-Run this code with *python quickstart.py* and you will see a web browser asking you for authentication. Click *Accept* and you are done with authentication. For more details, take a look at documentation: `OAuth made easy`_
+Run this code with *python quickstart.py* and you will see a web browser asking you for authentication. 
+Click *Accept* and you are done with authentication. For more details, take a look at documentation: `OAuth made easy`_
 
 .. _`APIs Console`: https://console.developers.google.com/iam-admin/projects
 .. _`OAuth made easy`: ./oauth.html
 
+Authentication via service accounts
```
Comment (shcheklein) [inline docs/quickstart.rst:39]: I think it's technically also part of the OAuth 2.0 - https://developers.google.com/identity/protocols/oauth2/service-account

File: docs/quickstart.rst [hunk 2]
```diff
@@ -29,11 +30,47 @@ Create *quickstart.py* file and copy and paste the following code.
...
+-----------------------------------
+
+Alternatively to OAuth2.0 client authentication *PyDrive2* can use Google Service Accounts for authentication.
+This type of authentication is useful when developing a servers side application which doesn't interact with endusers.
+
+1. create a service account in the Google Cloud Console
```
Comment (shcheklein) [inline docs/quickstart.rst:45]: Can we provide a link to Google docs here?

File: docs/quickstart.rst [hunk 3]
```diff
@@ -29,11 +30,47 @@ Create *quickstart.py* file and copy and paste the following code.
...
+Run this code with *python quickstart.py* and you will see a web browser asking you for authentication. 
+Click *Accept* and you are done with authentication. For more details, take a look at documentation: `OAuth made easy`_
 
 .. _`APIs Console`: https://console.developers.google.com/iam-admin/projects
 .. _`OAuth made easy`: ./oauth.html
 
+Authentication via service accounts
+-----------------------------------
+
+Alternatively to OAuth2.0 client authentication *PyDrive2* can use Google Service Accounts for authentication.
```
Comment (shcheklein) [inline docs/quickstart.rst:42]: Make Google Service Accounts a link to general docs about service accounts?

**Review by shcheklein — CHANGES_REQUESTED** [review]
Good stuff! Please consider a few minor things to improve.

File: docs/quickstart.rst (hunk 3 above)
Comment (tmotyl) [reply docs/quickstart.rst:42]: https://cloud.google.com/iam/docs/service-accounts

**Comment by tmotyl** [comment]
Hi @shcheklein 
unfortunately I will not be able to work on the pydrive in the foreseeable future, so feel free to adapt the PR as you wish.
Thanks for the nice contributor experience so far, and for maintaining the tool, I really appreciate that.

**Comment by shcheklein** [comment]
sure, @tmotyl - no worries at all ... I'll put need help and hacktoberfest tags and let's see if someone could take and complete this PR :) thanks for the contribution!

**Comment by shcheklein** [comment]
Closing this as stale.

**Pull request closed without merging**